### DISCUSSION TITLE
Localized Sitemap Not Generating Proper XML Format

### DISCUSSION THREAD

**Discussion by F-47** [vercel/next.js#69939] [Help]
### Provide environment information

```bash
Operating System:
  Platform: win32
  Arch: x64
  Version: Windows 11 Home
  Available memory (MB): 32561
  Available CPU cores: 12
Binaries:
  Node: 20.10.0
  npm: N/A
  Yarn: N/A
  pnpm: N/A
Relevant Packages:
  next: 14.2.6 // There is a newer version (14.2.8) available, upgrade recommended!
  eslint-config-next: 14.2.5
  react: 18.3.1
  react-dom: 18.3.1
  typescript: 5.5.4
Next.js Config:
  output: N/A
```

### Which example does this report relate to?

Sitemap

### What browser are you using? (if relevant)

Microsoft Edge

### How are you deploying your application? (if relevant)

Vercel

### Describe the Bug

I'm trying to implement the [localized sitemap](https://nextjs.org/docs/app/api-reference/file-conventions/metadata/sitemap#generate-a-localized-sitemap) as outlined in the Next.js documentation. The expected output should resemble the documented example (see below):

However, the actual output on my site is just a plain list of paths without the proper XML formatting.

Environment:
Next.js version: "14.2.6",
Translation package: "next-intl": "^3.17.2"

Question:
Could you provide any insights or solutions as to why this might be happening? I'm using next-intl for handling translations, and I'm wondering if that could be affecting the sitemap generation.

### Expected Behavior

The sitemap should be in the XML format as shown in the documentation, properly formatted with locale-specific URLs.

### To Reproduce

Implement a localized sitemap using the steps in the documentation. [Here](https://nextjs.org/docs/app/api-reference/file-conventions/metadata/sitemap#generate-a-localized-sitemap)
Deploy or run the app locally to inspect the sitemap at /sitemap.xml.
The sitemap is output as a plain list of paths, without any XML tags or formatting.

**Comment by archanaagivale30** [comment]
@F-47 , the sitemap generation looks fine if you look at it in the network tab. The issue is not with NextJs sitemap generation it's with xhtml:link tags in the browser. You can refer to the discussion for the same issue https://support.google.com/webmasters/thread/161147229/will-google-still-recognize-my-sitemap-if-i-change-the-declaration-protocol?hl=en

You can also refer to https://github.com/vercel/next.js/issues/67813 issue
